packages/terra-action-footer/src/ActionFooter.jsx

```diff
@@ -16,19 +16,25 @@ const propTypes = {
    * Actions to be displayed in the end socket
    */
   end: PropTypes.node,
+  /**
+   * Determines whether or not a border is shown
+   */
+  isBorderless: PropTypes.bool,
 };
 
 const defaultProps = {
   start: undefined,
   end: undefined,
+  isBorderless: false,
```

Comment:
What do people think about making this hasBorder with a default of true instead?

Reply:
Don't we typically want default our boolean props to false?